package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onnxjs-demo",
-  "version": "0.1.0",
+  "version": "0.1.6",
```

Comment:
do we want 1:1 correspondence with onnx.js in versioning ? Is that the idea ?

Reply:
yes I want to keep them consistent in future

Reply:
BTW, the version of `onnxjs-demo` is not that important because it is unlikely that anyone will consume `onnxjs-demo` as dependency and use its semversioning. The project is never published as a npm package.